runtime/queries/godot-resource/highlights.scm

```diff
@@ -0,0 +1,28 @@
+(section (identifier) @type.builtin)
+
+(attribute (identifier) @attribute)
+(property (path) @attribute)
```

Comment:
```suggestion
(property (path) @variable.other.member)
```

I think this matches a bit better based on other uses of property. What do you think?